app/date-select/controllers.test.js

```diff
@@ -0,0 +1,144 @@
+// const filters = require('../../common/controllers/filters')
```

Comment:
It would be good if we can tidy up this commit, maybe using an interactive rebase so that we don't need to worry about committing this approach to the history.

Reply:
this file doesn't even exist any more, so not sure how it showed up in the list of files for you

Reply:
It showed up when I went through the commits. It was attached to the early commit and then removed in a later one.

Reply:
Once this is ready maybe you can ensure it’s removed at the right commit and then it’ll never enter the history.

Reply:
Already done